prince-of-versions/src/main/java/co/infinum/princeofversions/helpers/parsers/JsonVersionConfigParser.java

```diff
@@ -140,7 +152,8 @@ public VersionContext parse(JSONObject data) throws JSONException {
                     this.currentVersion,
                     new VersionContext.Version(minVersion.toString()),
                     currentVersion.lessThan(minVersion),
-                    new VersionContext.UpdateContext(new VersionContext.Version(latestVersion.toString()), notificationType),
+                    new VersionContext.UpdateContext(new VersionContext.Version(latestVersion.toString()), notificationType, lastMinSdk,
```

Comment:
I concur, I've placed it here because I thought the UpdateContext contains all the optional parameters. I'll make the change.